docs/submit/osg-flock.md

```diff
@@ -1,97 +1,170 @@
-Configuring a submit host to flock to OSG
-=========================================
+Configure an HTCondor submit host to send excess jobs to OSG
+============================================================
+
+This document explains how to add a path for user jobs to flow from your local site out to the OSG,
+which in most cases means that the jobs will have far more resources available to run on than locally.
+If your local batch system frequently has many jobs waiting to run for a long time,
+adding a path to OSG may result in less waiting for your users.
```

Comment:
This intro pre-supposes that a site has local compute, which is not at required. We should also probably have a note or a couple of sentences pointing people to OSG Connect and why it may be a more reasonable solution.